src/main/java/com/iota/iri/conf/NetConfig.java

```diff
@@ -0,0 +1,627 @@
+package com.iota.iri.conf;
+
+import com.beust.jcommander.Parameter;
+import com.iota.iri.utils.IotaUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class NetConfig implements IotaConfig {
```

Comment:
`NetConfig` is a misleading name

Reply:
Hmm,
What name would you give it? `AbstractIotaConfig`? `AbstractConfig`?